### ISSUE TITLE
Adapt title

### ISSUE THREAD

**Issue by JJ** [geneura-papers/nowcasting-traffic#5]
Change the title. Discuss here or in the paper.

**Comment by unintendedbear** [comment]
I think that nowcasting traffic is not bad, but maybe we can add something, as in "Nowcasting traffic: -something else-".

**Comment by JJ** [comment]
Nowcasting traffic: predicting actual traffic from detected samples.

**Comment by amorag** [comment]
I propose:

Nowcasting traffic: predicting actual traffic from wireless detected samples.

**Comment by unintendedbear** [comment]
It is wireless indeed, but maybe that term gives the idea of detecting WiFi signals only. I know it doesn't have to be and literally BT is also wireless, but I don't know.

Bear with me T_T

**Comment by JJ** [comment]
"samples detected via Bluetooth and WiFi"

**Comment by unintendedbear** [comment]
Nowcasting traffic: predicting actual traffic from Bluetooth and WiFi detected samples (?)

**Comment by JJ** [comment]
_Nowcasting traffic: predicting number of vehicles from a sample detected via Bluetooth and WiFi_

**Comment by unintendedbear** [comment]
Sí, intentaba acortarlo, pero así está muy bien, me gusta. ¿Opiniones los demás? ¿Os parece bien?

**Comment by mgarenas** [comment]
done, I'll close the issue

**Comment by JJ** [comment]
Close by writing it on the actual paper and via the commit that commits
that, please.
